eng/UpdateAzureSdkGeneratorVersions.ps1

```diff
@@ -0,0 +1,7 @@
+param([string]$AutorestCSharpVersion, [string]$CadlEmitterVersion, [string]$SdkRepoRoot)
+
+Import-Module "$PSScriptRoot\UpdateGeneratorMetadata.psm1" -DisableNameChecking -Force;
+
+$SdkRepoRoot = Resolve-Path $SdkRepoRoot
+
+UpdateGeneratorMetaData $AutorestCSharpVersion $CadlEmitterVersion $SdkRepoRoot
```

Comment:
This script also looks identical with the first part of `eng/UpdateAzureSdkForNet.ps1`, should we call this script in that?